bitnami/grafana-operator/templates/grafana.yaml

```
@@ -65,9 +65,7 @@ spec:
       {{- end }}
   {{- if .Values.grafana.serviceAccount }}
   serviceAccount:
-  {{- with .Values.grafana.serviceAccount }}
-    {{- toYaml . | nindent 4 }}
-  {{- end }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.serviceAccount "context" $ ) | nindent 4 }}
```

@dgomezleon I just checked and the serviceAccount _is_ actually in the [grafana section].(https://github.com/bitnami/charts/blob/ca830ba72226e923335c49928810202a9129d440/bitnami/grafana-operator/values.yaml#L449)

```suggestion
    {{- include "common.tplvalues.render" ( dict "value" .Values.grafana.serviceAccount "context" $ ) | nindent 4 }}
```

---

You are right @arnecls I was checking [`grafana` values](https://github.com/bitnami/charts/blob/ca830ba72226e923335c49928810202a9129d440/bitnami/grafana/values.yaml#L568) instead.